rrmngmnt/host.py

```diff
@@ -211,14 +211,19 @@ def package_manager(self):
     def power_manager(self):
         return self.get_power_manager()
 
-    def executor(self, user=None, pkey=False, sudo=False):
+    def executor(self,
```

Comment:
same here, it this parameter doesn't belong here ... similar as the `pkey` is already depreciated as well for the same reason.